packages/manager/apps/telecom/src/app/telecom/pack/move/eligibility/address/move-eligibility-address.controller.js

```diff
@@ -170,18 +205,64 @@ export default class {
   /**
    * Retrieve buildings for the address
    */
-  searchBuildings() {
-    return this.OvhApiConnectivityEligibilitySearch.v6()
-      .searchBuildings(this.$scope, {
-        streetCode: this.address.street.streetCode,
-        streetNumber: this.address.streetNumber,
-      })
-      .then((data) => {
-        return data.result.length > 0 ? data.result : null;
-      })
-      .catch((error) => {
-        this.loading = false;
-        this.TucToast.error(error);
+  searchBuildings(copper) {
+    this.moveEligibilityAddressService
+      .searchBuildings(
+        this.address.streetNumber.hexacle,
+        this.address.street.streetCode,
+        this.address.streetNumber.number,
+      )
+      .then((response) => {
+        if (response.data.status === 'pending') {
```

Comment:
```suggestion
        if (response.status === 'pending') {
```

Reply:
Nope. It doesn't work like that. We have a response with data with status. I test it before make my PR